plugins/examples/__init__.py

```diff
@@ -558,6 +558,45 @@ def resolve_output(self, ctx):
 
         return types.Property(outputs)
 
+class CurrentSampleExample(foo.Operator):
+    @property
+    def config(self):
+        return foo.OperatorConfig(
+            name="example_current_sample",
+            label="Examples: Current Sample"
+        )
+    
+    def execute(self, ctx):
+        if ctx.current_sample is None:
+            return {"result": {"message": "No sample provided. Select a sample to see the result."}}
+        sample = ctx.dataset[ctx.current_sample]
+        current_group = None
+        group = None
+        group_field = ctx.dataset.group_field
```

Comment:
`dataset.group_field` isn't populated for dynamic groups unfortunately (the grouping could be more complex, eg an expression), so this would throw an error if run on a dynamic group view.

I think it is fine to just remove dynamic groups from this example